src/sentry/scripts/similarity/index.lua

```diff
@@ -631,6 +652,137 @@ local commands = {
+        appended to the existing data.
+        ]]--
+        function (configuration, arguments)
+            local entries = build_variadic_argument_parser({
+                {'index', identity},
+                {'key', identity},
+                {'data', function (value)
+                    return cmsgpack.unpack(value)
+                end}
+            })(arguments)
+
+            for _, entry in ipairs(entries) do
+                for band, data in ipairs(entry.data) do
+                    for _, item in ipairs(data) do
+                        local time, buckets = item[1], item[2]
+                        local expiration_time = get_index_expiration_time(
+                            configuration.interval,
+                            configuration.retention,
+                            time
+                        )
+                        local destination_bucket_frequency_key = get_bucket_frequency_key(
+                            configuration.scope,
+                            entry.index,
+                            time,
+                            band,
+                            entry.key
+                        )
+
+                        local touched = false
+                        for bucket, count in pairs(buckets) do
+                            local bucket_membership_key = get_bucket_membership_key(
+                                configuration.scope,
+                                entry.index,
+                                time,
+                                band,
+                                bucket
+                            )
+                            redis.call('SADD', bucket_membership_key, entry.key)
+                            redis.call('EXPIREAT', bucket_membership_key, expiration_time)
+
+                            redis.call(
+                                'HINCRBY',
+                                destination_bucket_frequency_key,
+                                bucket,
+                                count
+                            )
+                            touched = true
+                        end
+
+                        -- The destination bucket frequency key may have not
+                        -- existed previously, so we need to make sure we set
+                        -- the expiration on it in case it is new.
+                        if touched then
+                            redis.call(
+                                'EXPIREAT',
+                                destination_bucket_frequency_key,
+                                expiration_time
+                            )
+                        end
+                    end
+                end
+            end
+        end
+    ),
+    EXPORT = takes_configuration(
+        --[[
+        Exports data that is located at the provided ``index`` and ``key`` pairs.
+
+        Generally, this data should be treated as opaque method for extracting
```

Comment:
This way each input pair is associated with a single result. This is consistent with the responses from other commands, and also makes it so that if you do `EXPORT A B` and then need to `IMPORT A` and `IMPORT B` into different scopes that is possible without having to unpack the return value.